setup.py

```diff
@@ -1,28 +1,20 @@
 import setuptools
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
-
-
-with open("requirements.txt") as f:
-    requirements = f.read().splitlines()
-
-
 setuptools.setup(
-    name="greenplum-python",
-    version="1.0.0-beta1",
-    install_requires=requirements,
+    name="greenplum-python-fork",
+    version="1.0.0-beta4",
+    install_requires=["psycopg2==2.9.3"],
```

Comment:
toxwill run `setup.py`, correct? So maybe we don't need `requirements.txt` anymore.

Reply:
yes we can add __version__

Reply:
> yes we can add  “__version__”

Reply:
I think we'd better keep both of it.
Because `Python Guys` maybe just want to run locally they prefer to run `pip3 install -r requirements.txt`